runtime/calamari/src/xcm_config.rs

```diff
@@ -334,4 +339,7 @@ impl orml_xtokens::Config for Runtime {
 	type BaseXcmWeight = BaseXcmWeight;
 	type LocationInverter = LocationInverter<Ancestry>;
 	type MaxAssetsForTransfer = MaxAssetsForTransfer;
+	type MinXcmFee = AssetManager;
+	type MultiLocationsFilter = AssetManager;
+	type ReserveProvider = orml_traits::location::AbsoluteReserveProvider;
```

Comment:
nitpick: let's import this at the top of the file ? I do this sometimes as well, but feels inconsistent with the rest of the code.

Reply:
It relates to xcm. Not sure we really should move it to the `lib.rs`.

Reply:
Not move it to lib.rs, I mean:
`use orml_traits::location::AbsoluteReserveProvider;`
.
.
.
`type ReserveProvider = AbsoluteReserveProvider;`